src/components/ContactForm/index.jsx

```diff
@@ -52,25 +66,31 @@ export default function ContactForm({ submit }) {
 
         <div className="formInput">
           <label htmlFor="messageContent">
-            Message
+            {t("contactUs.contactUsForm.messageInputLabel")}
             <textarea
               className={getErrorClass(errors.message)}
               name="messageContent"
               id="messageContent"
-              placeholder="type your message here ..."
+              placeholder={t("contactUs.contactUsForm.messageInputPlaceholder")}
               aria-label="message"
               ref={register({
                 required: {
                   value: true,
-                  message: "You should write your message.",
+                  message: t(
+                    "contactUs.contactUsForm.messageInputErrorMessage0"
```

Comment:
notice how this requires the coder to know that error0 means required = not good. usually we can avoid naming things 0 1 2 etc